src/zenml/cli/stack.py

```diff
@@ -167,6 +176,14 @@ def register_stack(
                 name=model_deployer_name,
             )
 
+        if experiment_tracker_name:
```

Comment:
This is more of a reminder to make sure to extend the CLI commands for `zenml stack update`, `zenml <component> update` and `zenml <component> rename` when you merge @strickvl 's PR in (https://github.com/zenml-io/zenml/pull/497).

Reply:
Thanks, I just added that